approvals_tier_validation/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017-19 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Approvals Tier Validation",
+    "version": "13.0.1.0.0",
+    "category": "Tools",
+    "website": "https://github.com/OCA/server-ux",
+    "author": "Ecosoft, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["base_tier_validation"],
```

Comment:
You need to add a dependency with `mail` as well.

Reply:
@LoisRForgeFlow 
Thanks, if so, I will continue for the second iteration. And you guys can review it more.

Thanks!

Reply:
Great, would love to!